torchbox/torchbox.com
# Impact Report Page - BE

[Link to Ticket](https://torchbox.monday.com/boards/1192293412/pulses/1439824220)

### Description of Changes Made

- Adds a new `impact_reports` Django app
- Adds a new `ImpactReportPage` model, copied from https://github.com/torchbox/wagtail-torchbox/blob/4754f75f1f18d3072f705c4170f42deb182528fe/tbx/impact_reports/models.py#L31-L102
- Adds several StreamField blocks, also copied from the current build

> [!WARNING]  
> To avoid duplication of efforts, the template markup has not been written, as it is covered in the [FE ticket](https://torchbox.monday.com/boards/1192293412/pulses/1439824403).
> However, if required, I can add some basic markup for testing

### How to Test

- In Wagtail Admin, you should be able to add a new page of type **Impact report** (please see screenshots below)
 
### Screenshots

<details>
  <summary>Expand to see more</summary>

![Screenshot from 2024-03-27 11-26-49](https://github.com/torchbox/torchbox.com/assets/7713776/26739fe8-d2af-4783-9de8-8a8a5bce5ca8)

![image](https://github.com/torchbox/torchbox.com/assets/7713776/aa94e610-11bd-4b61-b96d-5d530972652a)

![image](https://github.com/torchbox/torchbox.com/assets/7713776/bb87f3e3-1c93-4736-a90b-2820dcfa9484)

</details>

### MR Checklist


#### Unit tests


#### Documentation


#### Browser testing

  - Latest version of Chrome on mac
  - Latest version of Firefox on mac
  - Latest version of Safari on mac
  - Safari on last two versions of iOS
  - Chrome on last two versions of Android

#### Data protection


#### Accessibility


#### Sustainability


#### Pattern library
